src/no_auth_required.js

```diff
@@ -1,3 +1,7 @@
 /***
  * @todo Redirect the user to main page if token is present.
- */
\ No newline at end of file
+ */
+
+if (localStorage.token!=undefined) {
+    window.location.href = '/';
+} 
```

Comment:
Similarly here, make a function and import them instead of importing these as header files.